optimism/FunctionSpace.py

```diff
@@ -193,7 +193,14 @@ def interpolate_to_points(functionSpace, nodalField):
     return jax.vmap(interpolate_to_element_points, (None, 0, 0))(nodalField, functionSpace.shapes, functionSpace.mesh.conns)
 
 
-def integrate_over_block(functionSpace, U, stateVars, dt, func, block,
+def vmapPropValue(propArray):
+    numAxes = len(propArray.shape)
+    if numAxes > 1:
+        return 0
+    else:
+        return None
```

Comment:
@lkgallup We should rewrite this function to use jax.lax.cond. It's a Jax if statement. It's kind of cryptic. Happy to help here.